modules/rtdModule/index.js

```diff
@@ -302,7 +302,8 @@ export const setBidRequestsData = timedAuctionHook('rtd', function setBidRequest
     }
   });
 
-  const shouldDelayAuction = prioritySubModules.length && _moduleConfig.auctionDelay && _moduleConfig.auctionDelay > 0;
+  const auctionDelay = _moduleConfig && _moduleConfig.auctionDelay;
```

Comment:
AuctionDelay wasn't reachable in some submodule  tests where rtd module wasn't configured